<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VmarkerGUI</class>
 <widget class="QMainWindow" name="VmarkerGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>322</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>VmarkerGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>498</width>
     <height>301</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>498</width>
     <height>301</height>
    </size>
   </property>
   <widget class="QGroupBox" name="grpSensor">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>160</y>
      <width>251</width>
      <height>136</height>
     </rect>
    </property>
    <property name="title">
     <string>Sensor Settings</string>
    </property>
    <widget class="QSlider" name="sldBright">
     <property name="geometry">
      <rect>
       <x>65</x>
       <y>110</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="sldGain2">
     <property name="geometry">
      <rect>
       <x>65</x>
       <y>50</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>31</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Gain 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Area</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spnBright">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>105</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spnGain1">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>15</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>15</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Gain 2</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spnArea">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>75</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Bright</string>
     </property>
    </widget>
    <widget class="QSlider" name="sldGain1">
     <property name="geometry">
      <rect>
       <x>65</x>
       <y>20</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>15</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSpinBox" name="spnGain2">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>45</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>31</number>
     </property>
    </widget>
    <widget class="QSlider" name="sldArea">
     <property name="geometry">
      <rect>
       <x>65</x>
       <y>80</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpCalibration">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>80</y>
      <width>251</width>
      <height>76</height>
     </rect>
    </property>
    <property name="title">
     <string>Calibration Settings</string>
    </property>
    <widget class="QSlider" name="sldCalTimeout">
     <property name="geometry">
      <rect>
       <x>65</x>
       <y>50</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>100</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spnCalSize">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>15</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>75</number>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Timeout</string>
     </property>
    </widget>
    <widget class="QSlider" name="sldCalSize">
     <property name="geometry">
      <rect>
       <x>65</x>
       <y>20</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>75</number>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSpinBox" name="spnCalTimeout">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>45</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>100</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpCommon">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>5</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Common Settings</string>
    </property>
    <widget class="QCheckBox" name="chkMouse">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Mouse</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkManCal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>45</y>
       <width>116</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Manual Calibration</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkMultiTouch">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>MultiTouch</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkAppleMac">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>45</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Apple Mac mode</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpLive">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>231</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Sensor Status</string>
    </property>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>15</y>
       <width>221</width>
       <height>156</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblCalStatus">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>175</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mouse status</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblCalPoint">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>185</y>
       <width>101</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="text">
      <string> </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdX">
     <property name="geometry">
      <rect>
       <x>155</x>
       <y>180</y>
       <width>64</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdY">
     <property name="geometry">
      <rect>
       <x>155</x>
       <y>205</y>
       <width>64</width>
       <height>26</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdSize">
     <property name="geometry">
      <rect>
       <x>155</x>
       <y>230</y>
       <width>64</width>
       <height>26</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lblCalStatus_2">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>185</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblCalStatus_3">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>210</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblCalStatus_4">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>235</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Size:</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Settings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCalibrate">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Calibrate</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>498</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
